robot_size: 0.2
max_vel: 0.22
rate: 20
vision_radius: 3.5
safety_distance:  0.4
collision_scanner_size: 50
